### PR TITLE
[No QA] Remove all inline named exports. Update style guide.

### PR DESCRIPTION
**Pull request by marcaaron**
cc @roryabraham 

### Details

Removes all inline named exports

### Fixed Issues
$ https://github.com/Expensify/App/issues/6011

### Tests / QA Steps
- Built storybook and ran tests to make sure changing those named exports did not cause issues.
- No specific test steps. Just normal regressions. Make sure app builds OK etc.

### Tested On

- [x] Web
- [ ] Mobile Web
- [ ] Desktop
- [ ] iOS
- [ ] Android

### CODE REVIEW

File: STORYBOOK.md [hunk 1]
```diff
@@ -48,11 +48,14 @@ export default {
...
 // Each story must be exported with a named export
-export const Default = Template.bind({});
+const Default = Template.bind({});
 Default.args = {
     text: 'Save & Continue',
     success: true,
 };
+export {
+    Default,
+}
```
Comment (aldo-expensify) [inline STORYBOOK.md:58]: Semi-colon missing after the `export` statement.

**Review by aldo-expensify — CHANGES_REQUESTED** [review]
Project built fine and looks fine.

NAB: Since all exports should be at the bottom of the file, including `default`, we should update other places of the documentation like (maybe is the only one :) ):

image

Update: Changed my comment into a NAB because I see that the code in all `xx.stories.js` don't have the default export at the end. Maybe you are intentionally leaving it out of the refactor?

File: STYLE.md [hunk 2]
```diff
@@ -264,7 +264,23 @@ ES6 provides two ways to export a module from a file: `named export` and `defaul
...
+export const somethingElse = 'stop';
+
+// Good
+const something = 'yep';
+const somethingElse = 'go';
+
+export {
+    something,
+    somethingElse,
+}
```
Comment (aldo-expensify) [inline STYLE.md:282]: Semi-colon missing after the `export` statement.

**Comment by marcaaron** [comment]
>NAB: Since all exports should be at the bottom of the file, including default, we should update other places of the documentation like (maybe is the only one :) ):

Honestly, I skimmed about half of the files and didn't see any so I think we can fix them as we go. 
Will fix the comment about the semi-colon.

**Review by aldo-expensify — DISMISSED** [review]
Looks good to me, it builds fine and I didn't notice anything no working in the app.

Leaving it un-merged in case @roryabraham wants to review.

**Review by aldo-expensify — APPROVED** [review]
Looks good to me!

**Review by roryabraham — APPROVED** [review]
LGTM

**Pull request merged**